plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/after/test/impl/CloseAllShellsExt.java

```diff
@@ -131,5 +131,10 @@ public void runBeforeShellIsClosed(Shell shell) {
 			return this.closedShellsTitles;
 		}
 	}
+	
+	@Override
+	public long getPriority() {
+		return -1000000;
```

Comment:
All constants should be be defined as constants (public static final). Also update the JavaDoc so that it is clear what's the priority.